trunk/SUAVE/Attributes/Solids/Rib.py

```diff
@@ -0,0 +1,30 @@
+# Rib.py
```

Comment:
What is rib?

Is this a component or a material?

Reply:
It's actually just Aluminum with an added minimum width. Suggest changing it to inherit from Aluminum rather than directly from Solid and adding the minimum width.